packages/mc-html-template/src/process-headers.ts

```diff
@@ -28,24 +32,33 @@ const mergeDirectives = (...directives: TDirective[]) =>
       ),
     {}
   );
-const toHeaderString = (directives: TDirective = {}) =>
-  Object.entries(directives)
+
+  return Object.entries(mergedDirectives)
     .map(
       ([directive, value]) =>
         `${directive} ${Array.isArray(value) ? value.join(' ') : value}`
     )
     .join('; ');
-const toStructuredHeaderString = (directives: TDirective = {}) =>
-  Object.entries(directives)
-    .map(
-      ([directive, value]) =>
-        `${directive}=${Array.isArray(value) ? value.join(' ') : value}`
-    )
-    .join(', ');
+};
+
+const parsePermissionsPolicyDirectives = (
```

Comment:
This is new: it now merges the `permissions-policy` default directives with the ones provided in the customization config.
This mimics the "merging behavior" that we have implemented in the MC Proxy.